src/Sulu/Bundle/ContentBundle/Resources/config/document.xml

```diff
@@ -97,6 +99,9 @@
 
         <service id="sulu_document_manager.document.subscriber.route"
                  class="Sulu\Component\Content\Document\Subscriber\RouteSubscriber">
+            <argument type="service" id="sulu_document_manager.document_manager"/>
+            <argument type="service" id="sulu_document_manager.document_inspector"/>
```

Comment:
In the latest document manager you can (and should) access the Inspector via. the `document_manager` itself.